docs/resources/serving_endpoint.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "MLflow"
+---
+# databricks_serving_endpoint Resource
+
+This resource allows you to manage [Model Serving Endpoints](https://docs.databricks.com/machine-learning/model-inference/serverless/serverless-real-time-inference.html) in Databricks.
+
+## Example Usage
+
+```hcl
+resource "databricks_serving_endpoint" "this" {
+  name = "ads-serving-endpoint"
+  config {
+    served_models {
```

Comment:
we don't have the aliasing on SDK level yet..

Reply:
Yeah I think we talked about adding this once aliasing is supported.

Reply:
what about customizing the schema, and simply doing (until aliases are natively supported):

```go
m["served_model"]=m["served_models"]
delete(m, "served_models")
```

?

Reply:
Yeah I can do that but I'll need to do something like this right since `served_models` is a nested field inside `config`: ```
ct := common.StructToSchema(endpoints.EndpointCoreConfigInput{},
    func(c map[string]*schema.Schema) map[string]*schema.Schema {
	    c["served_model"] = c["served_models"]
	    delete(c, "served_models")
	    return c
    })
    m["config"].Elem = ct
```
This doesn't quite work either since `Elem` maps originally to a `Schema` or is there a simpler way of doing this?